### ISSUE TITLE
Windows build is slow as molasses

### ISSUE THREAD

**Issue by CamiloMM** [mllrsohn/node-webkit-builder#195]
Ok, that's an exaggeration, but the startup time when I run NW against the current directory is like 1-2 seconds, and when I package it (and I made sure I'm not packing any file that's not used), it's like 6-8 seconds. After that it's fast enough, but startup time shouldn't be that big, even on an HDD.

Tested is Windows 7 x64.

**Comment by kessler** [comment]
It is because the way nw extracts itself to a folder in your computer's temp folder every time. As far as I know (but I did not research this subject recently) to avoid that you need to package and execute your application differently, so after it lands on a user's machine it is extracted only once and then you run it using nw.exe <dir>

**Comment by CamiloMM** [comment]
Yes, and that would not be a bad solution.

I mean, node-webkit-builder could have an option where it just copies over files and creates an "application folder" for each platform. Or bundle a zip file; I think NW supports reading a zip file faster than the way node-webkit-builder is doing it.

I'm much more concerned over startup time than having a single-file executable, and to be honest, if you're going to have a "single-file executable" and half-ass it by having two data files, DLLs and a locales folder, you might as well make a self-extracting archive. The current way is combining the bad part of both worlds.

**Comment by CamiloMM** [comment]
Either way, I want to say that the idea behind node-webkit-builder is great and it's really easy to use; testing it out was super easy but if I can shave off several seconds of startup time by coding something myself it's just a pity.

**Comment by felicienfrancois** [comment]
duplicates #164 ?

**Comment by CamiloMM** [comment]
Although I think there's a deeper issue here (unzipping shouldn't be that slow, ever), one simple solution is "not zipping". But really, there must be something wrong with whatever is unzipping the files. But I'm grateful for #200.

**Comment by adam-lynch** [comment]
Closing in favour of #164.
